src/persist-client/src/internal/state.proto

```diff
@@ -101,3 +101,8 @@ message ProtoStateRollup {
     // but now the value is a `struct HollowRollup`.
     map<uint64, string> deprecated_rollups = 12;
 }
+
+message ProtoRollup {
+    ProtoStateRollup state = 1;
+    repeated bytes diffs = 2;
```

Comment:
fairly strong preference for this to be repeated messages of seqno+bytes, so we can sanity check the seqnos on read

Reply:
the second ```repeated ProtoDiffAndSeqno diffs = 2;```